### PR TITLE
DEV-6655 Use benchmark-runner

### PR DESCRIPTION
**Pull request by mehmetscoskun**

### CODE REVIEW

File: performance/utils/loadFile.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-declare var window: Window;
+declare var window: any;
```
Comment (JosVerburg) [inline performance/utils/loadFile.ts:1]: If we can no longer use `Window` here than it means the `performance/tsconfig.json` is no longer used. We should either remove that one or make sure the benchmark runner uses it and than we can keep this typing.

File: package.json [hunk 2]
```diff
@@ -100,4 +101,4 @@
 		"typescript": "^3.8.3",
 		"uglify-js": "^3.8.0"
 	}
-}
+}
```
[the remark this answers is not in the document]
Comment (mehmetscoskun) [reply package.json:104]: done

[the remark this answers is not in the document]
Comment (mehmetscoskun) [reply package.json:104]: done

File: package.json [hunk 3]
```diff
@@ -18,8 +18,8 @@
 		"coverage": "nyc report --reporter=text-lcov | coveralls",
 		"integrationtests": "ts-mocha --require test/testhook.js \"test/specs/parsing/**/*.ts\" -p test/tsconfig.json",
 		"prepare": "npm run build",
-		"performance": "ts-node -P ./performance/tsconfig.json ./performance/runBenchmarks.ts",
-		"performance-server": "rimraf ./performance/lib && tsc -p ./performance/web.tsconfig.json && concurrently -k -p \"[{name}]\" -n \"tsc,rollup,server\" \"tsc -p ./performance/web.tsconfig.json -w\" \"rollup -c ./performance/rollup.config.js -w \" \"ts-node -P ./performance/tsconfig.json ./performance/server.ts\"",
+		"performance": "fonto-benchmark-runner --node",
```
[the remark this answers is not in the document]
Comment (mehmetscoskun) [reply package.json:21]: Should I do this for other two packages as well? fontoxpath-cache and whynot

File: package.json [hunk 4]
```diff
@@ -62,6 +62,7 @@
 	"license": "MIT",
 	"homepage": "https://github.com/FontoXML/fontoxpath",
 	"devDependencies": {
+		"@fontoxml/fonto-benchmark-runner": "0.0.1",
```
[the remark this answers is not in the document]
Comment (mehmetscoskun) [reply package.json:65]: done

File: package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (DrRataplan) [reply package.json:21]: Please do!

File: performance/utils/loadFile.ts (hunk 1 above)
Comment (JosVerburg) [reply performance/utils/loadFile.ts:1]: I think we can revert both changes is this file. i.e. `declare var window: Window` and `new Request(...`. 
That at least seems to be ok according to typescript. This should than be tested as part of the story to make the benchmarks run in the browser.

**Pull request merged**